app/build.gradle

```diff
@@ -36,7 +37,7 @@ android {
         compose true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion "1.4.3"
+        kotlinCompilerExtensionVersion compose_compiler_version
```

Comment:
If you want to go an extra mile you can move all the Deps and versions to a buildSrc and convert the groovy setup to gradle kotlin